docs/cdn-purge.md

```diff
@@ -0,0 +1,23 @@
+# Purging Cache on the CDN
+
+The majority of the requests, data served by Altis will be cached on the CDN to improve performance and delivery time to the user. In some situations you may need to invalidate specific URLs on the CDN. You can add your own CDN purge rule directly in PHP using `Altis\Cloud\purge_cdn_paths` helper function.
```

Comment:
```suggestion
The majority of the requests, data served by Altis will be cached on the CDN to improve performance and delivery time to the user. In some situations you may need to invalidate specific URLs on the CDN. You can add your own CDN purge rule directly in PHP using `Altis\Cloud\purge_cdn_paths()` helper function.
```

It's good practice to add a code example of this too so it's clear how to use that function.